seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/command/BaseTaskExecuteCommand.java

```diff
@@ -85,7 +85,7 @@ protected final void close(List<? extends Plugin<E>>... plugins) {
             for (Plugin<E> plugin : pluginList) {
                 try (Plugin<?> closed = plugin) {
                     // ignore
-                } catch (Throwable e) {
+                } catch (Exception e) {
```

Comment:
As discussed at #1530, catch `Throwable` is better.

Reply:
Throwable is the superclass of all errors and exceptions in Java. Error is the superclass of all errors, which are not meant to be caught by applications.

Reply:
Ok, make sense to me, we shouldn't catch `Error` in most of the time.